xapian-core/include/xapian/weight.h

```diff
@@ -29,6 +29,10 @@
 #include <xapian/types.h>
 #include <xapian/visibility.h>
 
+#include "registry.h"
```

Comment:
Should be `<xapian/registry,h>` and be added to the group above in sorted order.